sdk/storage/AzureStorageBlob/Source/DataStructures/Enumerations.swift

```diff
@@ -136,9 +136,10 @@ public enum TransferState: Int16 {
     /// Indicates whether the transfer is currently in a state that can be resumed.
     public var resumable: Bool {
         switch self {
-        case .paused, .failed:
+        // TODO: Switch .pending and .inProgress back to false eventually... but not now.
+        case .paused, .failed, .pending, .inProgress:
```

Comment:
Unless we can guarantee that the OS will have an opportunity to switch in progress or pending transfer to paused when the app crashes, transfers that should resume are not resuming. I haven't found a foolproof way to guarantee that.

Reply:
Makes sense to me. All transfers should be resumable unless explicitly block-listed.